05-cond.md

```diff
@@ -86,7 +86,7 @@ We can also combine tests using `and` and `or`.
 `and` is only true if both parts are true:
 
 ~~~ {.python}
-if (1 > 0) and (-1 > 0):
+if 1 > 0 and -1 > 0:
```

Comment:
@tbekolay why did you remove the parentheses? I think they improve readability even if they are not strictly necessary.